src/doc/man/cargo-metadata.adoc

```diff
@@ -134,7 +134,10 @@ The output has the following format:
                     /* Whether or not this target has doc tests enabled, and
                        the target is compatible with doc testing.
                     */
-                    "doctest": false
+                    "doctest": false,
+                    /* Whether or not this target should be built and run with `--test`
+                    */
+                    "tested": true
```

Comment:
This isn't quite correct.  Benchmarks also pass `--test`.  I would just say that this is a testable target (that is, one where `test = true` in the manifest).  I would also probably call this field `"test"` to match the naming convention in the manifest.

Reply:
done